wiki/plugins/images/templates/wiki/plugins/images/sidebar.html

```diff
@@ -3,15 +3,35 @@
 
 {% addtoblock "js" %}
 <script type="text/javascript">
-function insert_image(image_id) {
-  align=prompt('Enter an alignment. \'left\' or \'right\':', 'right');
-  caption=prompt('Enter a caption text. You may use markdown. You can edit the text after...');
-  imagetag = '\n[image:'+image_id+' align:'+align+']';
-  if (! caption == '') {
-    $('#id_content').insertAtCaret(imagetag+'\n    '+caption+'\n\n');
-  } else {
-    $('#id_content').insertAtCaret(imagetag+'\n\n');
-  }
+function insert_image(data) {
+    if (typeof data == "number") {
+        image_id = data;
+        $("#img_title").text(" {% trans "Insert Image" %} " + image_id);
```

Comment:
nitpicking - when you do translation inside JS, because some languages will have weird characters, it should be `{{ _("Insert Image")|escapejs }}`. It'll be picked up by `makemessages` as well :)